### DISCUSSION TITLE
KeePassXC corrupt database

### DISCUSSION THREAD

**Discussion by johannesbohn** [keepassxreboot/keepassxc#6464] [Q&A]
Hey guys! I have a problem.

I run version 2.5.4 of KeePassXC on MacOS

My KeePassXC database worked perfectly untill I installed an Electrum Wallet on my Mac. Suddenly, when I tried to open my database with my password (which i KNOW is the right one) it said: "Invalid credentials were provided, please try again. If this reoccurs, then your database file may be corrupt. (HMAC mismatch)" (se attached screenshot: 
<img width="515" alt="Skærmbillede 2021-04-29 kl  18 13 37" src="https://user-images.githubusercontent.com/83420743/116598896-b034c880-a927-11eb-8483-bba597e82b53.png">

I have not set anything besides a password to encrypt my database. No key file or Windows User Account.

I've tried to follow this advice: https://keepass.info/help/base/repair.html, where one has to create a new database and import the corrupt one into the new one, but it looks like it is only KeePass1-databases that can be imported? Mine is a .2-database.

I made a backup a while ago (or so I thought?!). I copied the file called "KeePass Password Database" onto an external hard drive. But I was not aware that one has to use plugins, triggers or a backup program... I tought a manual copy of the password database would be sufficient, but when I try to open one of these copies, I get the same error as you see in my screenshot...

I really need this databse back, as I have a number of extremely important passwords in there...

**Comment by droidmonkey** [comment]
Make sure you don't have a key file accidently entered into your credentials. Also recommend updating to KeePassXC 2.6.4.
